plutus-benchmark/validation/Common.hs

```diff
@@ -1,9 +1,7 @@
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TypeApplications  #-}
```

Comment:
Thanks for getting rid of these: I keep getting these zombie pragmas when I change code and forget to remove the things the compiler's told me to add.